pepper-apis/src/main/java/org/broadinstitute/ddp/service/AgeUpService.java

```diff
@@ -155,8 +155,8 @@ public int runAgeUpCheck(Handle handle, PexInterpreter interpreter, GovernancePo
                                     candidate.getParticipantUserId(),
                                     candidate.getParticipantUserId(),
                                     candidate.getParticipantUserGuid(),
-                                    policy.getStudyId(),
-                                    EventTriggerType.REACHED_AOM);
+                                    candidate.getParticipantUserGuid(),
```

Comment:
We should `null` the operator guid in the two call sites here as well!

Reply:
Nulled